cmd/kubeapps-apis/proto/kubeappsapis/core/plugins/v1alpha1/plugins.proto

```diff
@@ -11,24 +11,17 @@ import "protoc-gen-openapiv2/options/annotations.proto";
 // plugins in the future such as checking authz to the k8s api server using
 // the callers token.
 
-// Plugin
-//
-// A plugin can implement multiple services and multiple versions of a service.
-message Plugin {
-  option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_schema) = {
-    example: '{"name": "kapp_controller.packages", "version": "v1alpha1"}'
-  };
-  // Plugin name
-  //
-  // The name of the plugin, such as `fluxv2.packages` or `kapp_controller.packages`.
-  string name = 1;
-
-  // Plugin version
-  // 
-  // The version of the plugin, such as v1alpha1
-  string version = 2;
+service PluginsService {
```

Comment:
I thought the doc was only the core packaging api currently, but I guess you're re-ordering things here for consistency?

Reply:
Yep, that's precisely was my intention. Since they aren't functional changes, I assumed it was ok to send them along with this PR

Reply:
Yep, totally fine, just unexpected at first until I realised you'd just re-ordered.